src/js/bandwidth_test.js

```diff
@@ -157,6 +158,14 @@ function testVideoBandwidth(config) {
         rttStats.add(Date.parse(report.timestamp),
           parseInt(report.stat('googRtt')));
       }
+      if (report.type === 'ssrc') {
+        // This prevents it from collecting the intial send resolution which is
+        // something like 176x1407.
+        if (videoStats.length < 20) {
+          videoStats[0] = parseInt(report.stat('googFrameWidthSent'));
+          videoStats[1] = parseInt(report.stat('googFrameHeightSent'));
+        }
```

Comment:
Changed it to just grab the last reported resolution due to Chrome reporting bogus initial values. We need to add logic to ignore stats for N seconds which greatly depends on the device performance. Not sure if we want to do that now....